src/routes/error/NotFoundView.vue

```diff
@@ -0,0 +1,42 @@
+<script setup>
+import { useI18n } from 'vue-i18n';
+import { useThemedLogo } from '@/composables/themedLogo';
+const { t } = useI18n();
+const { logoSrc } = useThemedLogo();
+</script>
+
+<template>
+  <div class="pa-4">
+    <v-container>
+      <v-row>
+        <v-col cols="12" class="text-center">
+          <router-link to="/">
+            <img
```

Comment:
You need a task to abstract this in to a shared header and footer cross app, with a wrapper component functioning as the base layout so we can add header/footer easily to any page.